.github/workflows/test.yml

```diff
@@ -56,35 +51,24 @@ jobs:
           python-version: "3.8"
           channel-priority: strict
           activate-environment: dask-sql
-          environment-file: continuous_integration/environment-3.8-jdk11-dev.yaml
-      - name: Build the jar
+          environment-file: continuous_integration/environment-3.8-dev.yaml
+      - name: Build the Rust Datafusion bindings
```

Comment:
pedantic nit:

```suggestion
      - name: Build the Rust DataFusion bindings
```

Reply:
Good catch. I'll do a global find/replace as I'm sure I did this in several places.